stdpopsim/catalog/MusMus/species.py

```diff
@@ -0,0 +1,141 @@
+import stdpopsim
+
+from . import genome_data
+
+# Chromosome-specific recombination rate
+_recombination_rate = {
+    "1": 2.75145215530419e-09,
+    "2": 3.31510028178352e-09,
+    "3": 3.51437699680511e-09,
+    "4": 3.56171480378189e-09,
+    "5": 3.86074685482261e-09,
+    "6": 3.43642611683849e-09,
+    "7": 3.15498882608124e-09,
+    "8": 4.33723938517729e-09,
+    "9": 4.87408610885459e-09,
+    "10": 4.28382272762676e-09,
+    "11": 5.70613409415121e-09,
+    "12": 4.32252701579385e-09,
+    "13": 5.07783234828935e-09,
+    "14": 4.19354838709677e-09,
+    "15": 5.82580833090591e-09,
+    "16": 7.61865541027489e-09,
+    "17": 6.3034188034188e-09,
+    "18": 7.13171383998217e-09,
+    "19": 1.45096731154103e-08,
+    "X": 2.72149984880556e-09,
+    "Y": 0,
+    "MT": 0,
+}
+
+# Generic and chromosome-specific ploidy
+_species_ploidy = 2
+_ploidy = {
+    "1": _species_ploidy,
+    "2": _species_ploidy,
+    "3": _species_ploidy,
+    "4": _species_ploidy,
+    "5": _species_ploidy,
+    "6": _species_ploidy,
+    "7": _species_ploidy,
+    "8": _species_ploidy,
+    "9": _species_ploidy,
+    "10": _species_ploidy,
+    "11": _species_ploidy,
+    "12": _species_ploidy,
+    "13": _species_ploidy,
+    "14": _species_ploidy,
+    "15": _species_ploidy,
+    "16": _species_ploidy,
+    "17": _species_ploidy,
+    "18": _species_ploidy,
+    "19": _species_ploidy,
+    "X": _species_ploidy,
+    "Y": 1,
+    "MT": 1,
+}
+
+# Generic and chromosome-specific mutation rate
+_overall_rate = 5.4e-09
+_mutation_rate = {
+    "1": _overall_rate,
+    "2": _overall_rate,
+    "3": _overall_rate,
+    "4": _overall_rate,
+    "5": _overall_rate,
+    "6": _overall_rate,
+    "7": _overall_rate,
+    "8": _overall_rate,
+    "9": _overall_rate,
+    "10": _overall_rate,
+    "11": _overall_rate,
+    "12": _overall_rate,
+    "13": _overall_rate,
+    "14": _overall_rate,
+    "15": _overall_rate,
+    "16": _overall_rate,
+    "17": _overall_rate,
+    "18": _overall_rate,
+    "19": _overall_rate,
+    "X": _overall_rate,
+    "Y": _overall_rate,
+    "MT": 3.7e-08,
```

Comment:
@nspope Thank you so much for trying to help me figure this out! So I'm seeing some errors/prompts that I figured I should run by you before I muck this up again. It doesn't matter if I work from the same directory/git repo I cloned to the local machine I'm working on or if I start from scratch and replicate the commands you used above, when I get to `git push origin add_mus_mus` I see first this message:

```
To https://github.com/peterdfields/stdpopsim
 ! [rejected]        add_mus_mus -> add_mus_mus (non-fast-forward)
error: failed to push some refs to 'https://github.com/peterdfields/stdpopsim'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Then if I run `git pull`, this:
```
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint:
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint:
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
```
As for vscode, I'm normally working in vim and I wasn't seemingly getting the syntax right for the rebase changes so I thought I'd try the IDE route and failed there, too. Sorry for the confusion my pull request is causing and thank you again for your help.

Reply:
No problem @peterdfields -- guess my instructions were incomplete. The issue is that once you've rebased your local copy of the repository, it disagrees with your remote copy. If there were changes that you wanted keep from both local and remote repositories, you'd need to reconcile these manually (by pulling down the remote copy, fiddling around with the files where git detects conflicts, and committing those changes). That's not the case here-- the local copy is the only one you want to keep.

First, make sure you're on the feature branch: `git checkout add_mus_mus`. Then make sure you don't have any unstaged/uncommited changes sitting around: `git status` (this won't be the case if you've just cloned the remote repository, so you're working with a "fresh" local copy). 

Next -- assuming that there's no uncommited changes to files -- make sure that your rebased 'add_mus_mus' has the right commits; and that these are up-to-date with upstream/main. To do so, use `git log` to see the commit history ... the most recent commit should be 
```
commit e9a86c33cd4d702224a280778e0eaa0a8262baa0 (HEAD -> add_mus_mus)
Author: peterdfields <peter.fields@unibas.ch>
Date:   Sat Jan 21 21:51:29 2023 -0500

    ran maintenance script to pull ensembl info
```
and 8 commits below that should be the last update to upstream/main, which is
```
commit 7540e2e68bc3fa9d046baa1b05cb5e78b0a36999 (upstream/main, main)
Author: Peter Ralph <petrel.harp@gmail.com>
Date:   Mon Jan 23 09:33:58 2023 -0800

    update flake8 link, closes #1433 (#1434)
    
    * update flake8 link, closes #1433
```
It's always a good idea to check against the first few characters of the commit hash, because these will be unique, whereas the commit descriptions can be duplicated.

If everything looks OK, then you can force-push to overwrite your remote copy of the repository with your local changes.
```
git push origin add_mus_mus --force
```

Reply:
So I just ran `git push origin add_mus_mus --force` though I don't see any changes in the test success below. Here is top most output for `git log`:

```
commit cde4fd0f4cff7ee30725c6eea6a3c1cdc0fbe99c (HEAD -> add_mus_mus, origin/add_mus_mus)
Merge: b8d7f02 7540e2e
Author: peterdfields <peter.fields@unibas.ch>
Date:   Wed Feb 1 16:14:03 2023 -0500

    Merge branch 'popsim-consortium:main' into add_mus_mus

commit b8d7f027f0707c67e96a2721685828a4b887e361
Author: peterdfields <peter.fields@unibas.ch>
Date:   Sat Jan 21 21:51:29 2023 -0500

    ran maintenance script to pull ensembl info

commit bbe11cd5f42ba6675876e73135d177a12a47aba4
Author: peterdfields <peter.fields@unibas.ch>
Date:   Sat Jan 21 21:51:29 2023 -0500

    ran maintenance script to pull ensembl info

commit b6c5f272b9ce41df5c558f9cf5165ec4f115fdbd
Author: peterdfields <peter.fields@unibas.ch>
Date:   Tue Jan 31 16:08:56 2023 -0500

    Update setup.cfg

commit 7a0796b77ad9b253acfad0b6051bfc9aca94f428
Author: peterdfields <peter.fields@unibas.ch>
Date:   Mon Jan 30 18:16:48 2023 -0500

    ran pytest, fixed errors

commit b10772e13a1093d0466f090c83571ee4efc3e275
Author: peterdfields <peter.fields@unibas.ch>
Date:   Mon Jan 30 17:37:37 2023 -0500

    added species parameters for MusMus, fix errors

commit 7540e2e68bc3fa9d046baa1b05cb5e78b0a36999 (upstream/main, origin/main, origin/HEAD, main)
Author: Peter Ralph <petrel.harp@gmail.com>
Date:   Mon Jan 23 09:33:58 2023 -0800

    update flake8 link, closes #1433 (#1434)
```

And `git status`:

```
On branch add_mus_mus
Your branch is up to date with 'origin/add_mus_mus'.

nothing to commit, working tree clean
```

And `git pull` gives me `Already up to date.` Are there any other `push` like commands I should try?

Reply:
Looks fine, you just need to fix the missing bracket in the mutation rate dict, that I flagged above (that's what is causing the tests to fail).

Reply:
Done! Fingers crossed...